Jenkinsfile

```diff
@@ -37,11 +37,15 @@ pipeline {
         archiveArtifacts artifacts: "${REPO}/**/pdf/*.pdf", fingerprint: true, allowEmptyArchive: true
       }
     }
-    // stage('Tests') {
-    //   steps {
-    //     runXmostest("${REPO}", 'tests')
-    //   }
-    // }
+    stage('Tests') {
+      steps {
+        viewEnv() {
+          dir("${REPO}/tests") {
```

Comment:
To use the python virtualenv that includes the dependencies in requirements.txt, you need a withVenv() closure here, along with the viewEnv() and dir() you already have.